pkg/k8s/constants.go

```diff
@@ -0,0 +1,8 @@
+package k8s
+
+const (
+	// ExtensionAPIServerAuthenticationConfigMapName is the name of the ConfigMap where authentication data for extension API servers is placed.
+	ExtensionAPIServerAuthenticationConfigMapName = "extension-apiserver-authentication"
```

Comment:
Many constants live in `pkg/k8s/labels.go`, even constants which are not labels.  I'd recommend putting these in that file.